test/helpers/lightning.ts

```diff
@@ -211,7 +217,7 @@ export const mineBlockAndSync = async ({
   blockHeight?: number
   newBlock?: number
 }) => {
-  await bitcoindClient.generateToAddress(newBlock, RANDOM_ADDRESS)
+  await bitcoindOutside.generateToAddress(newBlock, RANDOM_ADDRESS)
```

Comment:
can we find a better name than `Outside`? Perhaps `nonGaloyBitcoind` or `bitcoindOwnedByThirdParty` or something like that?

Reply:
We also use the outside word for lnd. Is the same context correct?